packages/sandcastle/src/SandcastleDialog.tsx

```diff
@@ -0,0 +1,31 @@
+import { Dialog, DialogHeading, DialogProps } from "@ariakit/react";
+import classNames from "classnames";
+import { ReactNode } from "react";
+import "./SandcastleDialog.css";
+import { Text } from "@stratakit/bricks";
+
+export function SandcastleDialogHeading({ children }: { children: ReactNode }) {
+  return (
+    <DialogHeading>
+      <Text variant="headline-sm">{children}</Text>
```

Comment:
Perhaps a bit picky, but can we add a close button to the top right of the dialogues?

<img width="581" height="87" alt="Image" src="https://github.com/user-attachments/assets/c7df2c97-aec3-48fc-b697-b630bbb5c59c" />